wf/ja-jp/Content/Topics/ReleaseNotes/Workflows/production.htm

```diff
@@ -303,38 +303,10 @@
-                                                            <div>
-                                                                <h3>Fixes in Connector Builder</h3>
-                                                                <div>
-                                                                    <h4>OKTA-351973</h4>
-                                                                    <p>In the <span class="wintitle">Authentication</span> dialog, when <span class="userinput">OAuth 2.0</span> was selected as the authentication type, a change to the selected option in the <span class="uicontrol">Client Ownership</span> dropdown field was not displayed.</p>
-                                                                </div>
-                                                            </div>
-                                                        </div>
-                                                        <div>
-                                                            <h2 class="relnotes">Workflows</h2>
-                                                            <div>
-                                                                <h3>Keys in object outputs</h3>
-                                                                <p>In object outputs, children keys are no longer mappable or draggable.</p>
-                                                            </div>
-                                                        </div>
-                                                        <div>
-                                                            <h2>Connectors</h2>
-                                                            <div>
-                                                                <h3>AWS SSO connector renamed </h3>
-                                                                <p>To align with the rebranding of AWS SSO, the AWS SSO connector cards and documentation have been changed  to AWS Multi-Account Access.</p>
-                                                                <p>See <a href="../../Workflows/connector-reference/awssso/awssso.htm" class="MCXref xref">AWS SSOコネクタ</a>.</p>
-                                                            </div>
-                                                        </div>
-                                                    </div>
-                                                    <h3 class="relnotes">Previous releases</h3>
-                                                    <p>For previous releases, see Okta Workflows: Version History.</p>
+                                                    <h1 class="relnotes">未翻訳のドキュメント</h1>
+                                                    <p>このページは、現在まだ翻訳されていません。最新のリリース・ノートについては、
+			<a href="https://help.okta.com/wf/en-us/Content/Topics/ReleaseNotes/Workflows/workflows-release-notes.htm">リリース・ノート（英語版）</a>

```

Comment:
this could be a link to the production release notes instead of the generic workflows release notes. Is this intentional? I think I have the same question for several links, so would like to know why we are not linking to the prod/preview page